test/compilable/ddoc_markdown_tables_verbose.d

```diff
@@ -0,0 +1,19 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o- -preview=markdown -transition=vmarkdown
+// POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh
+
+/*
+TEST_OUTPUT:
+----
+compilable/ddoc_markdown_tables_verbose.d(19): Ddoc: replace table '| this | that |'
```

Comment:
What does this mean?

Reply:
It's logging that it's converting the table that starts with the header row `| this | that |`. It would probably read better if s/replace/formatting/. I'll commit that change.